profiling/src/lib.rs

```diff
@@ -950,6 +1072,18 @@ unsafe extern "C" fn alloc_profiling_malloc(len: u64) -> *mut ::libc::c_void {
         return ptr;
     }
 
+    let stats = ALLOCATION_PROFILING_STATS.with(|cell| {
+        let allocations = cell.borrow();
+        allocations.track_allocation(len)
+    });
+
+    if !stats.0 {
+        return ptr;
+    }
+
+    let samples = stats.1;
+    let total_size = stats.2;
```

Comment:
As far as I can tell, these values are still unscaled. Are you planning to add the code for scaling them up to provide an estimate of true allocations?